ginivision/src/main/java/net/gini/android/vision/camera/ImageStack.java

```diff
@@ -165,13 +168,15 @@ public void setImages(@Nullable final List<Bitmap> bitmaps) {
         stackItem3.setBackgroundColor(Color.TRANSPARENT);
         badge.setText("");
         badge.setVisibility(INVISIBLE);
+        subtitle.setVisibility(INVISIBLE);
```

Comment:
Why not an `ifelse` for this? Something like: 
```
if (bitmaps == null || bitmaps.size() == 0) { 
// Here you make the badge and subtitle invisible
} else {
 // The remaining code
}
```

This way you would make it more clear and wouldn't have redundant code.